README.md

```diff
@@ -662,7 +662,7 @@ It is expected that most users will be using the `Public` URL, however, it is al
 
 ## Release Notes
 
-[Changelist](./CHANGELIST.md)
+[Releases](https://github.com/microsoft/ApplicationInsights-JS/releases)
```

Comment:
But I like having this reference. suggest changing the "Changelist" above to "Release Notes" perhaps